docs/README.md

```diff
@@ -66,6 +66,37 @@ The Documentation is separated into different parts by sub folders.
 To update the API documentation run the `updateAPI.sh` file.
 Note that you have to install both the Java and Python -doc dependencies.
 
+#### Linux
+
+1. Install [Sphinx](https://pypi.org/project/Sphinx/) and [sphinx_rtd_theme](https://pypi.org/project/sphinx-rtd-theme/).
+
+```bash
+pip install Sphinx
+pip install sphinx-rtd-theme
```

Comment:
we can combine all these pip installs into one line. put it on the step 3 in install python dependencies, above.